.github/workflows/dependencies-head.yml

```diff
@@ -33,6 +33,10 @@ jobs:
         python -m pip --no-cache-dir --quiet install --upgrade --pre .[test]
         python -m pip list
 
+    - name: List release candidates and alphas
+      run: |
+        python -m pip list | grep '[0-9]rc[0-9]\|[0-9]a[0-9]\|[0-9]b[0-9]'
```

Comment:
Want to simplify this regex (find a number followed by rc, a, or b and then another number).